### PR TITLE
BSD 3-Clause license and Fair Play rules

### PR DESCRIPTION
**Pull request by dalcinl**

### CODE REVIEW

File: docs/source/guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+   If you absolutely have to break the previous rule, please do not add
+   additional functions to the namespace, or modify the API of existing
+   functions. Having additional functions exposed in distributed versions is
+   confusing for users and developers alike.
+
+
+.. Local variables:
+.. fill-column: 79
+.. End:
```
Comment (leofang) [inline docs/source/guidelines.rst:120]: Q: Are these needed?

**Review by leofang — APPROVED** [review]
LGTM, left some nits

File: docs/source/guidelines.rst (hunk 1 above)
Comment (dalcinl) [reply docs/source/guidelines.rst:120]: It's a comment, I'm an Emacs user :-)

File: docs/source/guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+   The usual approaches to build binary Python wheels involve the embedding of
+   dependent shared libraries. While such an approach may seem convenient and
+   often is, in the particular case of MPI and mpi4py it is ultimately harmful
+   to end users. Embedding the MPI shared libraries would prevent the use of
+   external, system-provided MPI installations with hardware-specific
+   optimizations and site-specific tweaks.
+
+   The MPI Forum is currently discussing the standardization of a proposal for
+   an Application Binary Interface (ABI) for MPI. Such standardization will
```
[the remark this answers is not in the document]
Comment (dalcinl) [reply docs/source/guidelines.rst:86]: @jeffhammond Great opportunity for a self-citation!

[the remark this answers is not in the document]
Comment (dalcinl) [reply docs/source/guidelines.rst]: @leofang I've added citation for the MPI Forum GitHub issues and MPI ABI paper.

**Review by jeffhammond — APPROVED** [review]

**Review by leofang — APPROVED** [review]

**Pull request merged**